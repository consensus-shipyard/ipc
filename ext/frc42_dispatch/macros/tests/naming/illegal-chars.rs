use frc42_macros::method_hash;

fn main() {
	// should panic because the name contains illegal chars
	let _str_hash = method_hash!("Bad!Method!Name!");
}
