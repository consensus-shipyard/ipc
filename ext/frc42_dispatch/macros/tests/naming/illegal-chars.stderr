error: proc macro panicked
 --> tests/naming/illegal-chars.rs:5:18
  |
5 |     let _str_hash = method_hash!("Bad!Method!Name!");
  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: IllegalName(IllegalCharacters)
